lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -208,7 +208,7 @@ class RenderBlockingResources extends Audit {
   static async audit(artifacts, context) {
     const {results, wastedMs} = await RenderBlockingResources.computeResults(artifacts, context);
 
-    let displayValue = '';
+    let displayValue;
```

Comment:
these are inferred correctly, right? in tsc playground it seems to work out out OK but some of the assertions we're removing later on in jsdoc makes me think some of them were necessary at one point in time for our checkJs setup...

https://www.typescriptlang.org/play/#code/DYUwLgBAZg9jDcAoRBLKEAUBZAhmAFgHQBOOAdgCYwC2GAlBAHwQAMhArAwN6IR-RwIAXghcAHgC4IARgA0EAJ5SATAF9E65AGMYZAM4xQhYDADmGWDEIAvOkA